### PR TITLE
Tests are working for newly added backport providers

### PR DESCRIPTION
**Pull request by potiuk**
---
Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Target Github ISSUE in description if exists
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by potiuk** [comment]
Solving problem with freshly added backport packages (Yeah - we have a new one) found in #9149

**Review by mik-laj — APPROVED** [review]

File: backport_packages/setup_backport_packages.py [hunk 1]
```diff
@@ -35,9 +36,20 @@
...
+
+sys.path.insert(0, SOURCE_DIR_PATH)
+
+# isort:skip
+from setup import PROVIDERS_REQUIREMENTS  # noqa
+
+# isort:skip
+import tests.deprecated_classes  # noqa
+# isort:skip
+from backport_packages.import_all_provider_classes import import_all_provider_classes  # noqa
```
Comment (turbaszek) [inline backport_packages/setup_backport_packages.py]: Would you mind adding a comment why such order? One day someone may wonder about that

Comment (potiuk) [reply backport_packages/setup_backport_packages.py]: Good point. Added.

File: backport_packages/setup_backport_packages.py [hunk 2]
```diff
@@ -35,9 +36,19 @@
...
-from tests.deprecated_classes import HOOKS, OPERATORS, SECRETS, SENSORS, TRANSFERS
+MY_DIR_PATH = os.path.dirname(__file__)
+SOURCE_DIR_PATH = os.path.abspath(os.path.join(MY_DIR_PATH, os.pardir))
+AIRFLOW_PATH = os.path.join(SOURCE_DIR_PATH, "airflow")
+PROVIDERS_PATH = os.path.join(AIRFLOW_PATH, "providers")
+
+sys.path.insert(0, SOURCE_DIR_PATH)
+
+# those imports need to come after the above sys.path.insert to make sure that Airflow
+# sources are importable witout having to add the airflow sources to the PYTHONPATH before
```
Comment (turbaszek) [inline backport_packages/setup_backport_packages.py]:
```suggestion
# sources are importable without having to add the airflow sources to the PYTHONPATH before
```

**Review by turbaszek — APPROVED** [review]

**Pull request merged**